src/Symfony/Component/HttpKernel/EventListener/SessionListener.php

```diff
@@ -33,10 +34,12 @@ public function __construct(ContainerInterface $container)
         $this->container = $container;
     }
 
-    protected function getSession(): ?SessionInterface
+    public function onKernelRequest(GetResponseEvent $event)
```

Comment:
I don't see how this change fix anything?
In the previous implementation, the `AbstractSessionListener::onKernelRequest` calls the `getSession` method which setup the storage. Doing the same thing in an overriden `onKernelRequest` ends to the same behavior right?

Reply:
@jderusse no it does not. `getSession` is called inside the callable set in `Request::setSessionFactory`, not in `onKernelRequest`